dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -194,6 +196,22 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
         return *this;
     }
 
+    winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationBuilder AppNotificationBuilder::AddTextBox(hstring id)
+    {
+        ThrowIfMaxInputItemsExceeded();
+
+        m_textBoxList.push_back(TextBox{ id });
+        return *this;
+    }
+
+    winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationBuilder AppNotificationBuilder::AddTextBox(hstring id, hstring placeHolderText, hstring title)
+    {
+        ThrowIfMaxInputItemsExceeded();
+
+        m_textBoxList.push_back(TextBox{ id, true, placeHolderText, title });
```

Comment:
I recommend converting m_textBoxList to an std::vector<std::wstring> and constructing the input element here. This way, you can remove the TextBox struct completely and just call append in GetActions since you already have the constructed element.

Reply:
done